_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -943,6 +1066,24 @@ authentication. For example:
 
 Use these modules whenever possible. You should always use IAM roles in your own modules any time you need to provide access to the AWS API. Using static API credentials should be avoided whenever possible.
 
+[[cleanup_expired_certs_deployment]]
+=== Automatically remove expired SSL/TLS certificates from IAM
+Next, use the link:https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/modules/cleanup-expired-certs/README.adoc[`cleanup-expired-certs`]
```

Comment:
Note this link won't work until https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/pull/58 is merged as it contains the new documentation in AsciiDoc format.

Reply:
Merged now, so resolving.